mithril-stm/src/participant/initializer.rs

```diff
@@ -0,0 +1,148 @@
+use crate::bls_multi_signature::{SigningKey, VerificationKeyPoP};
+use crate::key_reg::{ClosedKeyReg, RegParty};
+use crate::participant::StmSigner;
+use crate::stm::{Stake, StmParameters};
+use crate::RegisterError;
+use blake2::digest::Digest;
+use digest::FixedOutput;
+use rand_core::{CryptoRng, RngCore};
+use serde::{Deserialize, Serialize};
```

Comment:
Can you reorganize the imports to be approximately in the following form (also you can reuse the path exported from `lib.rs` if you want):
> std imports (if any), line break, third party crates imports, line break, internal imports

This is what we more or less try to enforce in our crates, so no obligation but if you can do it in the other files that would be lovely !

```suggestion
use blake2::digest::Digest;
use digest::FixedOutput;
use rand_core::{CryptoRng, RngCore};
use serde::{Deserialize, Serialize};

use crate::bls_multi_signature::{SigningKey, VerificationKeyPoP};
use crate::key_reg::{ClosedKeyReg, RegParty};
use crate::participant::StmSigner;
use crate::stm::{Stake, StmParameters};
use crate::RegisterError;
```